modules/tensor_mechanics/include/vectorpostprocessors/InteractionIntegral.h

```diff
@@ -66,10 +66,13 @@ class InteractionIntegralTempl : public ElementVectorPostprocessor
    * @param aux_stress Auxiliary stress -- computed in this method
    * @param grad_disp Gradient of auxiliary displacement -- computed in this method
    * @param grad_disp Auxiliary strain (FGM) -- computed in this method
+   * @param aux_disp Auxiliary displacement (RZ) -- computed in this method
```

Comment:
Maybe explicitly say that the aux displacements are only nonzero for RZ. In general, they are actually nonzero all the time, but we just don't use them, right?

Reply:
Yes, different terms of the interaction integral make use of different fields.